packages/storybook/.storybook/manager-head.html

```diff
@@ -78,4 +78,74 @@
 
     window.WcpConsent && WcpConsent.init('en-US', cookieBannerId, initCallback, onConsentChanged, WcpConsent.themes.light);
   });
-</script>
\ No newline at end of file
+</script>
+
+<script>
+  function addUsefulLinks() {
+    const linkStyle = 'color: black; text-decoration: none; text-align: center; cursor: pointer; padding: 0rem 0.5rem;';
+    const imageStyle = 'height: 0.75rem; margin-top: 0.125rem;';
+    const textStyle = 'margin-left: 0.25rem; font-size: 0.75rem;';
+    const linkContentStyle = 'display: flex;';
+    
+    const sidebarHeader = document.getElementsByClassName('sidebar-header');
+    if (sidebarHeader.length === 0) {
+      // sidebar container has not loaded in the page yet, retry in 500ms
+      console.log('sidebar-header not loaded');
```

Comment:
Should we log this on console?

Reply:
nop, forgot to remove this one. good catch :)